test/e2e/components/cookie.js

```diff
@@ -1,30 +1,30 @@
 class Cookies {
-    constructor(driver) {
-        this._driver = driver;
-    }
+  constructor(driver) {
+    this._driver = driver;
+  }
 
-    async get(name) {
-        const cookies = await this._driver.manage().getCookies();
-        return new Cookie(cookies.find(cookie => {
-            return cookie.name === name;
-        }));
-    }
+  async get(name) {
+    const cookies = await this._driver.manage().getCookies();
+    return new Cookie(cookies.find((cookie) => {
+      return cookie.name === name;
+    }));
+  }
 }
 
 class Cookie {
-    constructor(cookie) {
-        if (cookie) {
-            this._value = cookie.value;
-            delete cookie.value;
-            Object.assign(this, cookie);
-        }
+  constructor(cookie) {
+    if (cookie) {
+      this._value = cookie.value;
+      delete cookie.value;
+      Object.assign(this, cookie);
     }
+  }
 
-    get value() {
-        return this._value ? JSON.parse(this._value) : undefined;
-    }
+  get value() {
+    return this._value ? JSON.parse(this._value) : undefined;
+  }
 }
 
 module.exports = {
-    Cookies: Cookies
+  Cookies: Cookies,
```

Comment:
in theorie zijn de trailing commas dus gesupporteerd voor object literals maar op andere plaatsen dan weer niet ...
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

dus ik zou het veiliger vinden om ze niet te zetten ...